client/src/main.jsx

```diff
@@ -1,11 +1,16 @@
 import { createRoot } from 'react-dom/client';
+import React from 'react';
 import App from './App';
+import ReactGA from 'react-ga';
 import './style.css';
 import './mobile.css';
 import { ApiErrorBoundaryProvider } from './hooks/ApiErrorBoundaryContext';
 
 const container = document.getElementById('root');
 const root = createRoot(container);
+const trackingId = process.env.GOOGLE_ANALYTICS_TRACKING_ID;
```

Comment:
Need to add GOOGLE_ANALYTICS_TRACKING_ID to /.env.example and make a copy as .env. Otherwise, it will break here. Could you add this and test it?

Reply:
updated .env.example and add tracking id to this document https://udy4s0suiic.larksuite.com/wiki/NIQgwS2wJifMS9kJVEDukNg1smg